SWLOR.Game.Server/Feature/PerkDefinition/ForcePerkDefinition.cs

```diff
@@ -680,5 +683,86 @@ private void ForceInspiration()
                 .RequirementCharacterType(CharacterType.ForceSensitive)
                 .GrantsFeat(FeatType.ForceInspiration3);
         }
+
+
+        private void ForceDeath()
+        {
+            _builder.Create(PerkCategoryType.ForceDark, PerkType.ForceDeath)
+                .Name("Force Death")
+
+                .AddPerkLevel()
+                .Description("Deals 30 damage, and gives yourself 40 temporary hp")
+                .Price(3)
+                .RequirementSkill(SkillType.Force, 25)
+                .RequirementCharacterType(CharacterType.ForceSensitive)
+                .RequirementCannotHavePerk(PerkType.Benevolence)
+                .GrantsFeat(FeatType.ForceDeath1)
+
+                .AddPerkLevel()
+                .Description("Deals 45 damage, and gives yourself 55 temporary hp")
+                .Price(3)
+                .RequirementSkill(SkillType.Force, 35)
+                .RequirementCharacterType(CharacterType.ForceSensitive)
+                .RequirementCannotHavePerk(PerkType.Benevolence)
+                .GrantsFeat(FeatType.ForceDeath2)
+
+                .AddPerkLevel()
+                .Description("Deals 60 damage, and gives yourself 70 temporary hp")
+                .Price(3)
+                .RequirementSkill(SkillType.Force, 45)
+                .RequirementCharacterType(CharacterType.ForceSensitive)
+                .RequirementCannotHavePerk(PerkType.Benevolence)
+                .GrantsFeat(FeatType.ForceDeath3);
+
+                
+                
+
+
+
+        }
+
+        
+
+         private void Tutaminis()
+        {
+            _builder.Create(PerkCategoryType.ForceUniversal, PerkType.Tutaminis)
+                .Name("Tutaminis")
+
+                .AddPerkLevel()
+                .Description("Increase your defense by 10")
+                .Price(3)
+                .RequirementSkill(SkillType.Force, 10)
+                .RequirementCharacterType(CharacterType.ForceSensitive)
+                .GrantsFeat(FeatType.Tutaminis1)
+
+                .AddPerkLevel()
+                .Description("increase your defense by 20")
+                .Price(4)
+                .RequirementSkill(SkillType.Force, 20)
+                .RequirementCharacterType(CharacterType.ForceSensitive)
+                .GrantsFeat(FeatType.Tutaminis2)
+
+                .AddPerkLevel()
+                .Description("increase your defense by 30.")
+                .Price(5)
```

Comment:
Why is this perk priced at 5 SP but subsequent ones are cheaper? In general, the levels should progressively become more expensive. Having a spike like this in the middle is a little confusing. If this is what you want, I would suggest changing the 5 to a 3 and make level 4 and 5 cost 4SP each.